libnavui-voice/src/main/res/layout/mapbox_sound_button_layout.xml

```diff
@@ -27,8 +27,7 @@
         android:layout_width="@dimen/mapbox_dimen_56dp"
         android:layout_height="@dimen/mapbox_dimen_56dp"
         app:elevation="0dp"
-        android:layout_margin="@dimen/mapbox_dimen_2dp"
-        android:padding="@dimen/mapbox_dimen_18dp"
+        android:padding="@dimen/mapbox_dimen_16dp"
```

Comment:
Same as above. Should this be part of https://github.com/mapbox/mapbox-navigation-android/pull/4219?